### PR TITLE
[BD-18] Optimize paver settings call

### PR DESCRIPTION
**Pull request by emzubair**
### [BD-18] Optimize paver settings call

Updated `get_django_setting` method now it accepts comma separated `django_settings` & returns values for those settings.

### CODE REVIEW

File: pavelib/assets.py [hunk 1]
```diff
@@ -763,9 +763,9 @@ def webpack(options):
     Run a Webpack build.
     """
     settings = getattr(options, 'settings', Env.DEVSTACK_SETTINGS)
-    static_root_lms = Env.get_django_setting("STATIC_ROOT", "lms", settings=settings)
+    result = Env.get_django_setting("STATIC_ROOT,WEBPACK_CONFIG_PATH", "lms", settings=settings)
```
Comment (ziafazal) [inline pavelib/assets.py:766]: We should pass a list of setting instead of comma separated values.

Comment (ziafazal) [inline pavelib/assets.py:766]: nit: rename it to `django_settings`

File: pavelib/assets.py [hunk 2]
```diff
@@ -763,9 +763,9 @@ def webpack(options):
...
+    static_root_lms, config_path = result.encode('utf-8').splitlines()
```
Comment (ziafazal) [inline pavelib/assets.py:767]: why we need to encode string to `utf-8`?

**Review by ziafazal — CHANGES_REQUESTED** [review]
@emzubair reviewed it.

File: pavelib/assets.py (hunk 2 above)
Comment (emzubair) [reply pavelib/assets.py]: On `edly-io` repo, `get_django_setting` was returning `unicode` encoded value, but it was updated in `edx/master`. I checked it after the comment so there is no need for `utf-8` anymore, updated the code.

File: openedx/core/djangoapps/util/management/commands/print_setting.py [hunk 3]
```diff
@@ -23,13 +23,14 @@ class Command(BaseCommand):
     def add_arguments(self, parser):
         parser.add_argument(
             'setting',
```
Comment (ziafazal) [inline openedx/core/djangoapps/util/management/commands/print_setting.py:25]: rename: `setting` to `settings_to_print`

File: openedx/core/djangoapps/util/management/commands/print_setting.py [hunk 4]
```diff
@@ -23,13 +23,14 @@ class Command(BaseCommand):
...
-            help='Specifies the setting to be printed.'
+            nargs='+',
+            help='Specifies the list of settings to be printed.'
         )
 
     def handle(self, *args, **options):
-        setting = options.get('setting')
+        setting_options = options.get('setting')
```
Comment (ziafazal) [inline openedx/core/djangoapps/util/management/commands/print_setting.py:31]: `settings_to_print = options.get('settings_to_print')`

File: openedx/core/djangoapps/util/management/commands/print_setting.py [hunk 5]
```diff
@@ -23,13 +23,14 @@ class Command(BaseCommand):
...
 
-        if not hasattr(settings, setting):
-            raise CommandError(u'%s not found in settings.' % setting)
-
-        print(getattr(settings, setting))
+        for option in setting_options:
```
Comment (ziafazal) [inline openedx/core/djangoapps/util/management/commands/print_setting.py:33]: `for setting in settings_to_print:`

**Review by ziafazal — CHANGES_REQUESTED** [review]
@emzubair completed second round of review.

**Review by ziafazal — APPROVED** [review]
LGTM

File: pavelib/utils/envs.py [hunk 6]
```diff
@@ -249,21 +249,22 @@ def get_django_setting(cls, django_setting, system, settings=None):
...
-                    "print_setting {django_setting} 2>{log_file}".format(
-                        django_setting=django_setting,
+                    "print_setting {django_settings} 2>{log_file}".format(
+                        django_settings=django_settings,
                         log_file=cls.PRINT_SETTINGS_LOG_FILE
                     )
                 ),
                 capture=True
             )
             return str(value).strip()
```
Comment (ziafazal) [inline pavelib/utils/envs.py:265]: @emzubair split logic should be placed here.

**Pull request closed without merging**